index.html

```diff
@@ -59,6 +59,222 @@ <h3>LET'S CONNECT</h3>
+    </div>
+</section>
+  <!-- about section -->
+<section class="abtme">
+    <h3>
+        About Myself
+      </h3>
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <h4>LET'S CONNECT</h4>
     
+    <ul class="abtmeicons">
+              <li>
+                <a href="https://twitter.com/MjononoRonald"
+                  ><img src="./Images/icons/twitter.svg" alt="twittericon"
+                /></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/ronald-mjonono-86365988/"
+                  ><img src="./Images/icons/linkedin.svg" alt="linkedicon"
+                /></a>
+              </li>
+              <li>
+                <a href="#"><img src="./Images/icons/myspace.svg" alt="" /></a>
+              </li>
+              <li>
+                <a href="https://github.com/mj-e-boc"
+                  ><img src="./Images/icons/github.svg" alt="githubicon"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./Images/icons/evangelist.svg" alt="evangelist"
+                /></a>
+              </li>
+            </ul>
+            <button class="getresume">
+                Get my resume
+            </button>
+            <ul class="lans">
+                <li>
+                    Languages
+                </li>
+                
+                <li>
+                    <img src="./Images/icons/unionpointingdown.svg" alt="union">
+                
+                </li>
+                
+            </ul>
+            <ul class="lansIcons">
+                <li>
+                    <img src="./Images/icons/javascript.svg" alt="jsicon">
+                </li>
+                
+                <p>JavaScript</p>
+            </ul>
+            <ul class="lansIcons">
+                <li>
+                    <img src="./Images/icons/html.svg" alt="htmlicon">
+                </li>
+           <li>HTML</li>     
+                
+            </ul>
+            <ul class="lansIcons">
+                <li><img src="./Images/icons/css.svg" alt="cssIcon"></li>
+                
+                <li>
+                    CSS
+                </li>
+            </ul>
+            <ul class="lans names">
+                <li>
+                   Frameworks
+                </li>
+                <li>
+                    <img src="./Images/icons/rightarrow.svg" alt="union">
+                </li>
+                
+            </ul>
+            <ul class="lans names">
+                <li>
+                   Skills
+                </li>
+                <li>
+                    <img src="./Images/icons/rightarrow.svg" alt="union">
+                </li>
+                
+                
+            </ul>
```

Comment:
- [ ] Good job implementing `ul/li` for the technology lists 🥇 However, as suggested by the reviewer [here](https://github.com/mj-e-boc/MJay-Portfolio/pull/2#discussion_r1145990612), please makes sure that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, **_each of them in a `li`_**. Right now you are using `ul` for every technology category. The same applies to the technologies listed under each category (such as "JavaScript," "HTML," and "CSS") ⚒ As an example, you can do something like below 🔽

```
<ul>
  <li>
    Languages
    // here you can do the same for "JavaScript," "HTML," and "CSS"
  </li>
  <li>Frameworks</li>
  <li>skills</li>
</ul>
```